src/ChannelServer/Skills/Swordsman/Bash.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using Melia.Channel.Network;
+using Melia.Channel.Skills.Base;
+using Melia.Channel.World;
+using Melia.Channel.World.Entities;
+using Melia.Shared.Const;
+using Melia.Shared.Util;
+using Melia.Shared.World;
+
+namespace Melia.Channel.Skills.Swordsman
+{
+	/// <summary>
+	/// Handler for the Swordsman skill Thrust.
```

Comment:
Is there much of a reason to create a handler for Bash when you don't even change the comments?^^ Remember, the skill handlers will most definitely not stay the way they are, and every single one we add will have to be updated later.

Reply:
Pretty sure that was a work in progress, I've tried to do a better job of updating comments. I had two classes left on the Skills branch, for the basic classes, which were Scout and Swordsman.